src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        System.out.println(firstBall.toString());
+        Ball secondBall = Lottery.getRandomBall();
+        System.out.println(secondBall.toString());
+        Ball thirdBall = Lottery.getRandomBall();
+        System.out.println(thirdBall.toString());
     }
```

Comment:
There are redundant empty lines after the class declaration and method signature. According to the checklist, we should not add empty lines in these places.